src/styles/block/nav.scss

```diff
@@ -0,0 +1,46 @@
+.nav {
+
+  &__list {
+    padding: 0;
+    margin: 0;
+    padding-inline-start: 0;
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    list-style-type: none;
+  }
+
+  &__link {
+    text-decoration: none;
+    display: flex;
+    padding: 23px 0;
+    color: $color-txt-main;
+    position: relative;
+  }
+
+  &__Item,
+  &__link:hover {
+    color: $color-interactiv;
+  }
+
+  &__Item:not(:first-child) {
+    margin-left: 20px;
+  }
+
+}
+
+.is-active {
+  color: $color-interactiv;
+
+  &:after {
```

Comment:
It's not bem i guess, you need a modifier here __link--active. We used is-active for tests